### PR TITLE
[FEATURE]: Design Forgot Password module added

### PR DESCRIPTION
**Pull request by Hemu21**
#### PR Description
[FEATURE]: Design Forgot Password module added as per given ui.

#### Type of Change
- [ ] Bug fix (change which fixes an issue)
- [x] New feature (change which adds functionality)
- [ ] Documentation update

#### Select your work-area
- [x] Frontend
- [ ] Backend
- [ ] Documentation
- [ ] Others

#### Fixes (mention the issue number which this fixes)
#62 

#### Checklist
- [x] I have updated my branch and synced it with the project's 'develop' branch before making this PR.
- [x] I have optimized the file changes.
- [ ] I have added a snapshot of my work example.
- [ ] I have made a PR to the project's develop branch.

#### Undertaking
- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] I have checked for plagiarism and assure its authenticity.
- [x] I have read and followed the code of conduct for this repository. I understand that violation of this undertaking may have legal consequences.

- [x] I Agree

### CODE REVIEW

**Comment by Hemu21** [comment]
@SB2318  can you please check and merge it.

**Review by SB2318 — COMMENTED** [review]
You have many issues, I don't want to share the design here.
Again focus on learning, take your time and contact me on Discord.

File: frontend/src/screens/auth/OtpScreen.tsx [hunk 1]
```diff
@@ -1,26 +1,159 @@
...
+    marginTop: '40%',
+    borderColor: 'white',
+    borderWidth: 3,
+    borderTopLeftRadius: 30,
+    borderTopRightRadius: 30,
+    backgroundColor: 'white',
+    position: 'relative',
+  },
+  logo: {
+    height: hp(14),
```
Comment (SB2318) [inline frontend/src/screens/auth/OtpScreen.tsx:84]: decrease height and width of email logo

File: frontend/src/screens/auth/OtpScreen.tsx [hunk 2]
```diff
@@ -1,26 +1,159 @@
...
+    justifyContent: 'space-between',
+    width: '100%',
+    marginTop: 20,
+    paddingHorizontal: 10,
+    position: 'absolute',
+    top: hp(20),
+    left: wp(5),
+  },
+  title: {
+    fontSize: 23,
```
Comment (SB2318) [inline frontend/src/screens/auth/OtpScreen.tsx:108]: * Decrease fontSize to 20
* marginLeft : hp(-5) ❌
   margin : hp(2) ✔️ 
 * position :''absolute' ❌
    position: 'relative' or remove it ✔️
* don't need top or left
* Give marginTop: 10

File: frontend/src/screens/auth/OtpScreen.tsx [hunk 3]
```diff
@@ -1,26 +1,159 @@
...
+    top: hp(-4),
+    left: wp(5),
+    fontWeight: 'bold',
+  },
+  otp: {
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    width: '100%',
+    marginTop: 20,
+    paddingHorizontal: 10,
```
Comment (SB2318) [inline frontend/src/screens/auth/OtpScreen.tsx:102]: * paddingHorizontal:4 to 6
 * marginHorizontal: 3
 * justifyContent: 'flex-start' or 'center'
* alignItems: 'center'

File: frontend/src/screens/auth/OtpScreen.tsx [hunk 4]
```diff
@@ -1,26 +1,159 @@
...
+      <View style={styles.innerContainer}>
+        <Image source={require('../../assets/mail.png')} style={styles.logo} />
+        <Text style={styles.title}>
+          We have sent you OTP to your email address for verification
+        </Text>
+        <View style={styles.otp}>
+          {error && <Text style={styles.error}>Invalid OTP</Text>}
+          <TextInput
+            style={!error ? styles.input : styles.input1}
+            maxLength={4}
```
Comment (SB2318) [inline frontend/src/screens/auth/OtpScreen.tsx:48]: maxLength ={6}

File: frontend/src/screens/auth/OtpScreen.tsx [hunk 5]
```diff
@@ -1,26 +1,159 @@
...
+    height: 60,
+    width: '80%',
+    fontSize: 24,
+    borderRadius: 10,
+    borderColor: PRIMARY_COLOR,
+    borderWidth: 3,
+    paddingHorizontal: 10,
+    marginLeft: hp(3),
+  },
+  input1: {
```
Comment (SB2318) [inline frontend/src/screens/auth/OtpScreen.tsx:127]: * fontSize: 20
* borderRadius:7
* paddingHorizontal:4
* borderWidth:1
* Remove Left Margin


same changes will applied to **input1**

**Review by SB2318 — APPROVED** [review]

**Pull request merged**